about.html

```diff
@@ -1,121 +1,110 @@
         </p>
-      </div>
-      <img src="./images/creative-commons-license-logo.jpeg" alt="logo" />
+        <img class="blogo" src="img/techcrunchlogo.png" alt="Tech Crunch logo">
     </section>
 
-    <section class="about-page2">
-      <div class="prg1">
-        <h3 class="about-program">See the past CC Global Summits</h3>
-        <hr class="underline" />
-      </div>
-      <div class="about-images">
-        <div class="imageleft">
-          <img src="./images/image1.jpg" class="photo" alt="Confrence 2021" />
-        </div>
-        <div class="overlay">
-          <p class="overlay-text">Conference 2022</p>
-        </div>
+    <section class="peventscontainer"><hr/>
+        <h2 class="peventshead">See past Microverse events</h2>
+        <p class="peventsdesc">Take a look at the revealing of the past three phases.</p>
+        <ul class="pevents">
+            <li class="pevent one">
+                <h3>2010</h3>
+                <p>Microverse 2018 in California</p>
+            </li>
+            <li class="pevent two">
+                <h3>2013</h3>
+                <p>Microverse 2019 in California</p>
+            </li>
+        </ul>
+    </section>
 
-        <div class="imageleft">
-          <img src="./images/image2.jpg" class="photo" alt="Conference 2022" />
+    <footer class="footer">
+        <div class="partners">
+            <h2 class="partnerhead">Partner</h2>
+            <div class="partnerlogo">
+                <img class="plogo" src="img/tech.png" alt="Tech Crunch logo">
+                <img class="plogo" src="img/EdSurgelogo.png" alt="EdSurge logo">
+                <img class="plogo" src="img/CincoDíaslogo.png" alt="Cinco Días logo">
+                <img class="plogo" src="img/Infobaelogo.png" alt="Infobae logo">
+            </div>
         </div>
-        <div class="overlay">
-          <p class="overlay-text">Conference 2022</p>
+        <div class="bfooter">
+           
+                <img class="blogo" src="img/techcrunchlogo.png" alt="Tech Crunch logo">
+           
+            <div class="bfootertext">
+                <h4>2022 Microverse. Some rights reserved.</h4>
+                <p>Tech Crunch in association with Microverse.</p>
+            </div>
         </div>
-      </div>
-    </section>
-    <div class="footer-class2">
-      <div class="prg1">
-        <h3 class="program">Partners</h3>
-        <hr class="underline" />
-        <a class="bold">mozilla</a>
-        <a class="faint">Google</a>
-        <a class="bold">NAVER</a>
-        <a class="faint">daumkakao</a>
-        <a class="bold"><i class="fa fa-airbnb"></i>airbnb</a>
-      </div>
-    </div>
-
-    <script src="./style.js" async defer></script>
-  </body>
+    </footer>
```

Comment:
- [ ] Kindly decrease the height of the footer a little bit in the `desktop version` to match the Behance design.

| Your design | Behance design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/168911253-a6b4d224-438e-4af0-8267-6378a035bea0.png) | ![image](https://user-images.githubusercontent.com/24703171/168911433-da33eab5-b38d-4328-9d78-487c6b1bd2fb.png) |

- [ ] Kindly change the `background color to black` and `color to white` in the `desktop version` in the `about` page for the footer to match the Behance guidelines.
![image](https://user-images.githubusercontent.com/24703171/168911821-a6b93bb6-6542-4267-8661-a1fd81173733.png)